src/send-query.js

```diff
@@ -0,0 +1,12 @@
+import ajax from './ajax';
+
+export default function sendQuery(query, config) {
+  const url = `https://${config.domain}/api/graph`;
+  const headers = {
+    authorization: `Basic ${btoa(config.apiKey)}`,
```

Comment:
oh okay .. in that case, you might want to import/use isomorphic btoa right? afaik, this would break in node. No?